src/PSRule.Rules.Azure/rules/Azure.VNET.Rule.ps1

```diff
@@ -243,6 +243,11 @@ Rule 'Azure.LB.AvailabilityZone' -Type 'Microsoft.Network/loadBalancers' -If { I
     }
 }
 
+# Synopsis: Load balancers should be deployed with Standard SKU for production workloads.
+Rule 'Azure.LB.StandardSKU' -Type 'Microsoft.Network/loadBalancers' -Tag @{ release = 'GA'; ruleSet = '2021_09'; } {
+    IsStandardLoadBalancer;
```

Comment:
By itself this would only return true which would not provide a good reason.

Nothing wrong with calling `IsStandardLoadBalancer` but maybe update function to remove `.Result` so the reason is returned. This should still work for the other pre-condition using this function.

Reply:
Ah good catch @BernieWhite. Fixed 🙂

Reply:
Ah I might change the `IsStandardLoadBalancer` output type to `PSRule.Runtime.AssertResult`.